src/psyclone/tests/domain/gocean/transformations/gocean1p0_transformations_test.py

```diff
@@ -1744,9 +1801,13 @@ def test_accloop(tmpdir):
     acclpt = ACCLoopTrans()
     accpara = ACCParallelTrans()
     accdata = ACCEnterDataTrans()
+    cbtrans = GOConstLoopBoundsTrans()
 
-    psy, invoke = get_invoke("single_invoke_three_kernels.f90", API, idx=0)
+    psy, invoke = get_invoke("single_invoke_three_kernels.f90", API, idx=0,
+                             dist_mem=False)
     schedule = invoke.schedule
+    # This test expects constant loop bounds
+    newsched, _ = cbtrans.apply(schedule, {"const_bounds": True})
```

Comment:
pylint complains that newsched is unused (and again at line 1867)

Reply:
Done